zerver/webhooks/uptimerobot/doc.md

```diff
@@ -0,0 +1,36 @@
+Receive Zulip notifications for your Uptime Robot monitors!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+    Make sure to specify the topic in the URL above. Otherwise, the
+    default topic `General` will be used.
+
+1. Go to **My Settings** on Uptime Robot, and select **Add Alert Contact**.
+   Set **Alert Contact Type** as webhook.
+
+1. Set **Friendly Name** to a name of your choice, such as `Zulip`. Set **URL to notify** to the
+   URL constructed above with a **&** added at the end. Select **Send as JSON (application/json)**
```

Comment:
What's the story with needing an extra `&` at the end?  We should just add that `&` to the template if it's needed.

Reply:
Changed the template accordingly.